### PR TITLE
MDEXP-253-Fix-Metadata Fix data Migration issue on goldenrod

### PR DESCRIPTION
**Pull request by andrii-novytskyi**
## Purpose
https://issues.folio.org/browse/MDEXP-253

## Approach
After changing the mechanism of adding default profiles from API calls to DB scripts, we should add metadata to our default profiles data(previously it was created by RMB when we called our API to create profiles).

- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Comment by andrii-novytskyi** [comment]
**Profiles before changes:**
image
image
**Logs after installing new version of module:**
image
**Profiles after installing new version of module:**
image
image

File: src/main/resources/templates/db_scripts/schema.json [hunk 1]
```diff
@@ -70,12 +70,12 @@
     {
       "run": "after",
       "snippetPath": "default_data/create_default_mapping_profile.sql",
-      "fromModuleVersion": "mod-data-export-2.1.0"
+      "fromModuleVersion": "mod-data-export-2.2.0"
```
Comment (KVupp) [inline src/main/resources/templates/db_scripts/schema.json:73]: let this be 2.1.4, our next release version

Comment (andrii-novytskyi) [reply src/main/resources/templates/db_scripts/schema.json]: done)

**Comment by KVupp** [comment]
Andrii, the test that you did was it upgrade? If so which version to which version?

**Comment by andrii-novytskyi** [comment]
@KVupp 
image
 the same versions

**Comment by KVupp** [comment]
so that is the last master version to current version. Can we please test it for fameflower version(1.1.1) to current version as well

**Comment by andrii-novytskyi** [comment]
ok. i will do it

**Comment by KVupp** [comment]
Also please use this story https://issues.folio.org/browse/MDEXP-253

**Comment by andrii-novytskyi** [comment]
ok, then I will close this PR and create another one after finishing with a test

**Comment by KVupp** [comment]
> ok, then I will close this PR and create another one after finishing with a test

I don't think that's necessary, just edit the title, the branch name doesnt matter

**Comment by VictorSoroka1** [comment]
I think commit messages though should be changed to contain the actual ticket number so we will have the link in JIRA.

**Comment by KVupp** [comment]
> I think commit messages though should be changed to contain the actual ticket number so we will have the link in JIRA to the PR.

The links in JIRA are created with the title for the PR as far as I know. The commit messages don't matter, I usually don't use ticket numbers for commit

**Pull request merged**